tavla/src/Board/scenarios/Table/components/Spinner.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react'
+export const CircularCountdown = ({
+    duration = 10000,
+    size = '1.5em',
+    strokeWidth = 3,
+    color = 'var(--warning-color)',
+}) => {
```

Comment:
Vi kun bruker 10000 som duration, og aldri sender inn size eller strokeWidth - hva tenker du om å droppe å ha de som props? Personlig litt for å ikke støtte flere props enn det som er nødvendig 🤓